tools/accuracy_checker/README.md

```diff
@@ -76,6 +76,21 @@ If you see the error about directory/file not found, please try remove manually
 pip install --upgrade --force-reinstall .
 ```
 
+#### Running the tool inside IDE for development purposes
+
+Accuracy Checker tool has entry point for running in CLI, however majority of popular code editors or IDE expect scripts as starting point of application.
+Sometimes it can be useful to have opportunity to run the tool as script for debugging or enabling new models.
+For usage Accuracy Checker inside the IDE, you need create script in accuracy_checker root directory (e.g. `<open_model_zoo>/tools/accuracy_checker/main.py`)
```

Comment:
```suggestion
Accuracy Checker tool has entry point for running in CLI, however majority of popular code editors or IDE expects scripts as starting point of application.
Sometimes it can be useful to have opportunity to run the tool as script for debugging or enabling new models.
For usage Accuracy Checker inside the IDE, you need to create a script in accuracy_checker root directory (e.g. `<open_model_zoo>/tools/accuracy_checker/main.py`)
```